packages/esm-sms-app/src/config-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  configurationPageSize: {
+    _type: Type.Number,
+    _description: 'The default page size for the configuration table',
+    _default: 10,
+  },
+  deliveryTime: {
+    _type: Type.Array,
+    _description:
+      'The default delivery times after which test messages are sent.They as a delay before sending the test messages, i.e After 2hrs, After 30 mins',
+    _default: [],
+  },
+  smsLogsColumns: {
+    _type: Type.Array,
+    _description: 'Columns to display in logs table. Each value should be a property of SMSlogs record type',
```

Comment:
How do I find out what the properties of the SMSlogs record type are? You should either provide a link in the description, or use the [`oneOf`](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#oneof) validator.